compose/canary_tx.sh

```diff
@@ -0,0 +1,234 @@
+#!/usr/bin/env bash
```

Comment:
Suggestion: Add some comment describing what the intended purpose of this script is?